cmd/openshift-install/create.go

```diff
@@ -313,14 +314,15 @@ func waitForConsole(ctx context.Context, config *rest.Config, directory string)
 			if silenceRemaining == 0 {
 				logrus.Debug("Still waiting for the console route...")
 				silenceRemaining = logDownsample
+				err = errors.New("no routes found in openshift-console namespace")
```

Comment:
Hrm.  I feel like there's a cleaner way to catch `wait.Until` timeouts, but don't have an example to point to at the moment.  I'll look into this a bit more and see if I can come up with something.

Reply:
:+1:

Reply:
Ah, how about the pattern from [here][1]?  For you, it would be something like

```go
err = consoleRouteContext.Err()
if err != nil && err != context.Canceled {
	return errors.Wrap(err, "waiting for openshift-console URL")
}
```

instead of master's current:

```go
if err != nil {
	return "", errors.Wrap(err, "waiting for console route to be created")
}
```

block entirely.

[1]: https://github.com/openshift/installer/pull/786/commits/1642eaf0711d735975794643e94f64d25cc9163c

Reply:
:+1: testing that now :)